pixi.toml

```
@@ -40,6 +40,8 @@ docs = "cd docs && make html"
 readthedocs = "rm -rf $READTHEDOCS_OUTPUT/html && cp -r docs/_build/html $READTHEDOCS_OUTPUT/html"
 
 [feature.test.dependencies]
+boto3 = "*"
```

```suggestion
boto3 = ">=1.40.55"
```
they switched to `noarch` at some point.